src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -770,9 +770,9 @@ def test_vm_disk_max_shares_etc(self, resource_group):
         self.cmd('sig create -g {rg} --gallery-name {g1}')
         self.cmd('sig image-definition create -g {rg} --gallery-name {g1} --gallery-image-definition image --os-type linux -p publisher1 -f offer1 -s sku1 --features "IsSecureBootSupported=false IsMeasuredBootSupported=false" --hyper-v-generation V2', checks=[
             self.check('features[0].name', 'IsSecureBootSupported'),
-            self.check('features[0].value', 'false'),
+            self.check('features[0].value', 'false', False),
```

Comment:
why the change

Reply:
Service change. It returns "False" not "false"

Reply:
can the test util be changed to support case-insensitive compare?

Reply:
The "False" I added means case sensitive = False. Default value is True.